src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -734,14 +757,15 @@ private void DrawPatchLinks()
     /// <summary>
     ///   Clears the map and rebuilds all nodes
     /// </summary>
-    private void RebuildMapNodes()
+    private void RebuildMap()
     {
         foreach (var node in nodes.Values)
-        {
             node.Free();
-        }
 
         nodes.Clear();
+
+        RebuildRegionConnections();
+
         connections.Clear();
```

Comment:
I have just one thing I can't wrap my head around: why is this called here and then immediately the connections list is cleared? If the connection list rebuilt in _Draw ends up with different data, that data is then never used by RebuildRegionConnections except on the first render when the player patch is focused.

Reply:
I've fixed this by calling it after connections have been rebuilt